.changeset/tiny-rivers-build.md

```diff
@@ -0,0 +1,5 @@
+---
+'@graphql-tools/federation': patch
+---
+
+Handle shared subscription root fields correctly
```

Comment:
@ardatan can you please improve the writing of this changeset? Maybe share the error it's fixing and other info?

Reply:
Added more details about the bug